robottelo/datafactory.py

```diff
@@ -179,9 +179,9 @@ def valid_emails_list():
             gen_string('alphanumeric'),
             gen_string('alphanumeric'),
         ),
-        u'"():;"@example.com',
         u'!#$%&*+-/=?^`{|}~@example.com',
     ]
+    return [email.replace('"', '').replace('`', '') for email in email_list]
```

Comment:
Well the question is if such emails will ever be used by anyone, though if you guys feel strongly about this, I'll pull it back, no big deal